fullmetrics_dash.py

```diff
@@ -8,73 +8,80 @@
 ## Arguments to run the script
 
 parser = argparse.ArgumentParser()
-parser.add_argument("--verbosity", type=int, help="increase output verbosity, use --verbosity 1 to enable")
 parser.add_argument("integration", type=str,
                     help="specify the integration namespace for your dashboard. Ex: mysql, ntp, system")
+parser.add_argument("--site", "-s", type=str, default="us", choices=["us", "eu"], 
+	help="The site to send data, us (datadoghq.com) or eu (datadoghq.eu), default: us")
+parser.add_argument("--verbosity", type=int, help="increase output verbosity, use --verbosity 1 to enable")
 args = parser.parse_args()
 integration = args.integration
 
 ## Keys and timeframe
 
-api_key = "xxx"
-app_key = "xxx"
+api_key = "<API KEY>"
+app_key = "<APP KEY>"
 timestamp = int(time.time()) - 86400
 
-
+if str(args.site) == "eu":
+	api_host = "https://api.datadoghq.eu/"
+	dashboard_site = "https://app.datadoghq.eu"
+else:
+	api_host = "https://app.datadoghq.com/"
+	dashboard_site = "https://app.datadoghq.com"
 ## Init
 
 options = {
-    'api_key': str(api_key),
-    'app_key': str(app_key)
+    'api_key': str(api_key),	
+    'app_key': str(app_key),
+	'api_host': str(api_host)
 }
 
 initialize(**options)
 
 ## Metrics endpoint
 
-r = requests.get("https://app.datadoghq.com/api/v1/metrics?api_key="+ api_key + "&application_key=" + app_key + "&from="+str(timestamp)+"")
+r = requests.get(api_host + "api/v1/metrics?api_key="+ api_key + "&application_key=" + app_key + "&from="+str(timestamp)+"")
 
 r.json()
 metrics_list = []
 
-print("init: ", metrics_list)
+print "init: ", metrics_list
```

Comment:
I'm a little hesitant on this being switched to python2 since that was EOL'd 

https://www.python.org/doc/sunset-python-2/ 

I think it's better to keep it Python3